lib/IDE/TypeReconstruction.cpp

```diff
@@ -1451,6 +1494,11 @@ static void VisitNodeSetterGetter(
     }
   }
 
+  if (!type_result.HasSingleType()) {
+    result._error = "bad type";
```

Comment:
Do we have a testcase for this case/diagnostic?

Reply:
No. These are invalid cases that should never come up (but maybe if your swiftmodule doesn’t match your debuginfo or something? I don’t know)

Reply:
I don't think we should worry too much (if we'll ever run across a case where this happens, we can consider adding a test), in particular considering that, as you say, this code might go away in the next while.